examples/logs/plot_snakes.log

```diff
@@ -0,0 +1,27 @@
+Please be patient. Learning will take 5-20 minutes.
```

Comment:
simply to preserve them as a snapshot of current behavior. But we can get rid of them.